skimage/feature/register_translation.py

```diff
@@ -132,6 +159,9 @@ def register_translation(src_image, target_image, upsample_factor=1,
         Defines how the algorithm interprets input data.  "real" means data
         will be FFT'd to compute the correlation, while "fourier" data will
         bypass FFT of input data.  Case insensitive.
+    return_error : bool, optional
```

Comment:
would ``return_residual`` be clearer?

Reply:
I'm not sure whether we have other functions returning residual errors, if yes we should be consistent with names.